src/main/resources/application.properties

```diff
@@ -46,6 +46,7 @@ waitlist.resend.fixed.rate: 15000
 failed.notifications.collection.name: failed_notifications
 failed.notifications.collection.ttl: 600
 notification.retry: 3
+notification.httpRequest.timeout:3000
```

Comment:
Follow the Spring property naming rules recommendations from this page that we use:
https://docs.spring.io/spring-boot/docs/2.1.6.RELEASE/reference/html/boot-features-external-config.html

In the Spring documentation they recommend this:
"We recommend that, when possible, properties are stored in lower-case kebab format, such as my.property-name=acme."